TabulateSmarterTestContentPackage/Tabulator.cs

```diff
@@ -1425,7 +1439,9 @@ string ValidateContentAndWordlist(ItemContext it, XmlDocument xml)
                     {
                         if (node.NodeType == XmlNodeType.CDATA)
                         {
-                            ValidateContentCData(it, node, termIndices, terms);
+                            var imgList = ValidateContentCData(it, node, termIndices, terms);
```

Comment:
It doesn't feel right to have `ValidateContentCData` return an image list.  The job of `ValidateContentCData` is to do the validation and it turns out part of that validation is the image alt tags.  So I think it should be included in there.  It looks like you would need to pass in `xml` to the `ValidateContentCData` method and then you can still call `ReportMissingImgAltTags` but let's call it from within `ValidateContentCData`